README.md

```diff
@@ -69,24 +95,20 @@ import type { TemporalState } from 'zundo';
 const useTemporalStore = <T,>(
   selector: (state: TemporalState<StoreState>) => T,
   equality?: (a: T, b: T) => boolean,
-) => useStore(originalStore.temporal, selector, equality);
-```
-
-## Then bind your components
-
-Use your store anywhere, including `undo`, `redo`, and `clear`!
+) => useStore(useStoreWithUndo.temporal, selector, equality);
```

Comment:
For consistency in the docs, I replaced all references to stores created with `temporal` with `useStoreWithUndo`. I think this could makes things more clear to readers, instead of referring to the store by multiple names like `useStoreWithUndo`, `originalStore`, `useStoreA`, `withTemporal`, etc.

Reply:
Thanks, I didn't realize I used a different name each time. Oops!